### PR TITLE
enable ui module publication

### PR DESCRIPTION
**Pull request by Nexucis**
This PR is here to (re) enable the publication of the package `codemirror-promql` and `lezer-promql` under the new npm org `prometheus-io`

Following the last news on the CI/CD of Prometheus, I choose to create a new github action to do it. Hope it's fine like that.

I choose to publish the libs under the tag v0.XX.YY to follow the same thought around the Prometheus Go module. 

Signed-off-by: Augustin Husson <husson.augustin@gmail.com>

### CODE REVIEW

**Comment by Nexucis** [comment]
For the release procedure, I'm still unsure what to do regarding the changelog. Should we use the Changelog of Prometheus or use and keep a specific changelog for codemirror-promql / lezer-promql.

**Comment by Nexucis** [comment]
once this PR is ok and merged, we will be able to archive the repositories `prometheus/lezer-promql` and `prometheus/codemirror-promql`

File: RELEASE.md [hunk 1]
```diff
@@ -144,6 +144,12 @@ Entries in the `CHANGELOG.md` are meant to be in this order:
 * `[ENHANCEMENT]`
 * `[BUGFIX]`
 
+Then bump the UI module version
```
Comment (juliusv) [inline RELEASE.md:147]:
```suggestion
Then bump the UI module version:
```

**Comment by juliusv** [comment]
Would be good if someone a little more familiar with the GitHub actions build stuff like maybe @roidelapluie could have a brief look as well.

**Comment by Nexucis** [comment]
I added the NPM token in the secrets. We are ready to publish the packages in the prometheus-io organization.

**Comment by roidelapluie** [comment]
Can we merge this @juliusv ?

**Comment by juliusv** [comment]
:+1: Yep, looks fine to me as far as I understand it, assuming you are good with it then too :)

**Pull request merged**